internal/command/jsonformat/plan.go

```diff
@@ -224,12 +229,29 @@ func (plan Plan) renderHuman(renderer Renderer, mode plans.Mode, opts ...plans.Q
 		}
 
 		if importingCount > 0 {
+			if forgettingCount > 0 {
```

Comment:
I did this for the non-console formatter, I wasn't sure how well it would play with all the rendering code here so I opted for this method. I can make a follow up to try to clean this up if that works?